src/main/java/com/walmartlabs/x12/asn856/Item.java

```diff
@@ -46,6 +48,14 @@
      * LIN: Item Identification
      */
     private List<LINItemIdentification> itemIdentifications;
+    /*
+     * REF: references
+     */
+    private List<REFReferenceInformation> refList;
+    /*
+     * DTM: Date/Time Reference
+     */
+    List<DTMDateTimeReference> dtmReferences;
```

Comment:
private

Reply:
will add to next PR along with some attributes missed on other loop objects